src/Sulu/Bundle/ContactBundle/Tests/Functional/Controller/ContactControllerTest.php

```diff
@@ -1877,20 +1873,22 @@ public function testDelete()
         $contact = $this->createContact('Max', 'Mustermann');
         $this->em->flush();
 
-        $client = $this->createTestClient();
-        $client->request('DELETE', '/api/contacts/' . $contact->getId());
+        $contactId = $contact->getId();
+
+        $client = $this->client;
+        $client->request('DELETE', '/api/contacts/' . $contactId);
 
         $this->assertHttpStatusCode(204, $client->getResponse());
 
-        $client = $this->createTestClient();
-        $client->request('GET', '/api/contacts/' . $contact->getId());
+        $client = $this->client;
```

Comment:
Would also not be necessary. If we would always be using `$this->client` this error also would not happen :see_no_evil: